src/pages/observability-packs.js

```diff
@@ -47,7 +43,35 @@ const ObservabilityPacksPage = ({ data, location }) => {
   const [querySearch, setQuerySearch] = useQueryParam('search', StringParam);
   const [queryFilter, setQueryFilter] = useQueryParam('filter', StringParam);
   const [querySort, setQuerySort] = useQueryParam('sort', StringParam);
-
+  const [view, setView] = useState(VIEWS.GRID);
+  useInstrumentedData(
```

Comment:
_super_ nitpcik: do you mind adding some blank lines between these hooks? It would be nice to match the style of the other components as closely as we can.

Reply:
yes of course, i've tidied them up and grouped similar ones, let me know if there's anything else needs massaging @zstix